sklearn/MultiOneVsRest.py

```diff
@@ -0,0 +1 @@
        Data.

        Returns
        -------
        y : dict of [sparse array-like], shape = {predictors: n_samples}
          or {predictors: [n_samples, n_classes], n_predictors}.
            Predicted multi-class targets across multiple predictors.
            Note:  entirely separate models are generated
            for each predictor.
        """
        # check to see if the fit has been performed
        check_is_fitted(self, 'estimators_')
        
        results = {}
        for label, model_ in self.estimators_.iteritems():
            results[label] = model_.predict( X)
        return(results)
    
    def predict_proba(self, X):
        """Probability estimates. This returns prediction probabilites 
        for each class for each label in the form of 
        a dictionary. 
       
        Parameters
        ----------
        X : array-like, shape = [n_samples, n_features]
        Returns
        -------
        prob_dict (dict) A dictionary containing n_label sparse arrays 
        with shape = [n_samples, n_classes]. 
        Each row in the array contains the the probability of the 
        sample for each class in the model,
        where classes are ordered as they are in `self.classes_`.
        """
        # check to see whether the fit has occured. 
        check_is_fitted(self, 'estimators_')
        
        results ={}
        for label, model_ in self.estimators_.iteritems():
            results[label] = model_.predict_proba(X)
        return(results)

    def score(self, X, Y):
        """"Returns the mean accuracy on the given test data and labels.
       
        Parameters
        ----------
        X : array-like, shape = [n_samples, n_features]
        Y : (sparse) array-like, shape = [n_samples, p_targets]
           
        Returns
        -------
        scores (np.array) Array of p_target floats of the mean accuracy 
        of each estimator_.predict wrt. y.   
        """
        check_is_fitted(self, 'estimators_')
        # Score the results for each function
        results =[]
        for i in range(self._num_y):
            estimator = self.estimators_[i]
            results.append(estimator.score(X,Y[:,i]))
        return results

    def get_params(self):
        '''returns the parameters of the estimator.''' 
        return self.estimator.get_params()

    def set_params(self, params):
        """sets the params for the estimator."""
        self.estimator.set_params(params)

    def __repr__(self):
        return 'MultiOneVsRestClassifier( %s )' %self.estimator.__repr__()
    
    @property
    def multilabel_(self):
        """returns a vector of whether each classifer is a  multilabel
        classifier in tuple for
        """
        return [(label, model_.multilabel_) for label, model_ in self.estimators_.iteritems()]
    @property
    def classes_(self):
        return [(label, model_.label_binarizer_) for label, model_ in self.estimators_.iteritems()]

```

Comment:
Or you could just copy over the code to `multioutput.py`, remove this file and force push because that is what you are ultimately going to do anyway.